### ISSUE TITLE
Client parser ERROR

### ISSUE THREAD

**Issue by mtmk** [nats-io/nats.net.v2#341]
Seen this error a couple of times now. Need to investigate further.
```
Client parser ERROR, state=0, i=0: proto='"\nPING\r\nPING\r\nPING\r\nPING\r\nPING\r\nP"...'
```

2.1.0-preview.3 / 2.10.9

1) start server
2) dotnet run
3) restart server

```csharp
var nats = new NatsConnection();

Task.Run(async () =>
{
    await foreach (var msg in nats.SubscribeAsync<string>("x")) { }
});

while (true)
    await nats.PublishAsync("x", "x");
```

note: works with v2.0.2

**Comment by caleblloyd** [comment]
Here's a packet capture

[invalid-protocol-operation.zip](https://github.com/nats-io/nats.net.v2/files/14013703/invalid-protocol-operation.zip)

**Comment by mtmk** [comment]
resolved by #347
